packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -1,208 +1,56 @@
+/* eslint-disable no-use-before-define */
+
 import React from 'react';
-import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { fade, withStyles } from '@material-ui/core/styles';
-import ButtonBase from '@material-ui/core/ButtonBase';
-import { setRef, withForwardedRef } from '@material-ui/core/utils';
-import { clamp } from '@material-ui/lab/utils';
-
-export const styles = theme => {
-  const commonTransitionsOptions = {
-    duration: theme.transitions.duration.shortest,
-    easing: theme.transitions.easing.easeOut,
-  };
-
-  const trackTransitions = theme.transitions.create(
-    ['width', 'height', 'transform'],
-    commonTransitionsOptions,
-  );
-  const thumbTransitions = theme.transitions.create(
-    ['transform', 'box-shadow'],
-    commonTransitionsOptions,
-  );
-
-  const colors = {
-    primary: theme.palette.primary.main,
-    disabled: theme.palette.grey[400],
-    thumbOutline: fade(theme.palette.primary.main, 0.16),
-  };
+import { useTheme, withStyles, fade, lighten } from '@material-ui/core/styles';
+import { useForkRef, ownerWindow } from '@material-ui/core/utils';
+import { chainPropTypes } from '@material-ui/utils';
+import ThumbLabel from './ThumbLabel';
```

Comment:
The spec calls this "value label" would it make sense to follow suit?

Reply:
Yeah OK, both options are valid. My guess is that value label is better when considering the slider as a whole. I will change it.